src/components/NumberGenerator.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useState } from "react";
+import random from "lodash.random";
+import { Button } from "@mui/material";
+
+interface State {
+  prevNumber: null | number;
+  currentNumber: null | number;
+  pickedNumbers: number[];
+}
+
+const NumberGenerator = () => {
+  const [state, setState] = useState<State>({
+    prevNumber: null,
+    currentNumber: 0,
+    pickedNumbers: [],
+  });
+
+  const generateNumber = useCallback(() => {
+    const randomNumber = random(1, 90);
```

Comment:
constant values should be in a config file

Reply:
resolved